src/PrimDist.hs

```diff
@@ -186,38 +193,38 @@ instance Show ErasedPrimDist where
   show (ErasedPrimDist d) = show d
 
 -- | Draw a value from a primitive distribution in the @Sampler@ monad
-sample ::
+draw ::
      PrimDist a
   -> Sampler a
```

Comment:
If we're calling sampling drawing now, it should return a `Drawer` not a `Sampler`, but I suspect it's smarter to just not call it `draw`